org.jasp.JASP.json

```diff
@@ -0,0 +1,430 @@
+{
+	"app-id": "org.jasp.JASP",
+	"runtime": "org.kde.Sdk",
+	"runtime-version": "5.10",
+	"sdk": "org.kde.Sdk",
+	"sdk-extensions": [
+		"org.freedesktop.Sdk.Extension.gfortran-62"
+	],
+	"command": "org.jasp.JASP",
+	"finish-args":[
+		"--socket=x11",
+		"--socket=wayland",
+		"--share=ipc",
+		"--share=network",
+		"--filesystem=home",
+		"--env=PATH=/usr/bin:/app/bin:/usr/lib/sdk/gfortran-62/bin"
+	],
+	"build-options": {
+		"cflags": "-O2",
+		"cxxflags": "-O2",
+		"env": {
+			"PATH": "/usr/bin:/app/bin:/usr/lib/sdk/gfortran-62/bin"
+		}
+	},
+	"cleanup": [
+		"/include"
+	],
+	"modules":[	
+		{
+			"name": "boost",
+			"buildsystem": "simple",
+			"build-commands":[
+				"./bootstrap.sh --with-libraries=filesystem,system",
+				"./b2 -j4 install --prefix=/app" 
+			],
+			"sources":[
+				{
+					"type": "archive",
+					"url": "https://dl.bintray.com/boostorg/release/1.64.0/source/boost_1_64_0.tar.gz",
+					"sha256": "0445c22a5ef3bd69f5dfb48354978421a85ab395254a26b1ffb0aa1bfd63a108"
+				}
+			]
+		},
+		{
+			"name": "r",
+			"config-opts": ["--enable-R-shlib", "--disable-BLAS-shlib"],
+			"build-options": {
+				"ldflags": "-L/usr/lib/sdk/gfortran-62/lib"
+			},
+			"sources": [
+				{
+					"type": "shell",
+					"commands": [ "/usr/lib/sdk/gfortran-62/install.sh" ]
+				},
+				{
+					"type":"archive",
+					"url": "https://cran.r-project.org/src/base/R-3/R-3.4.4.tar.gz",
+					"sha256":"b3e97d2fab7256d1c655c4075934725ba1cd7cb9237240a11bb22ccdad960337"
+				}
+			]
+		},
+		{
+			"name": "nloptr",
+			"build-options": {
+				"build-args": ["--share=network"]
+			},
+			"sources": [
+				{
+					"type":"archive",
+					"url": "https://cran.r-project.org/src/contrib/nloptr_1.0.4.tar.gz",
+					"sha256":"84225b993cb1ef7854edda9629858662cc8592b0d1344baadea4177486ece1eb"
+				},
+				{
+					"type": "shell",
+					"commands": 
+						[ 
+							"echo \"all:\n\t\n\" > Makefile", 
+							"echo \"install:\n\tR CMD INSTALL ./\" >> Makefile" 
+
+						]
+				}
+			]
+		},
+		{
+			"name": "igraph",
+			"config-opts": ["--disable-graphml"],
+			"build-options": {
+				"build-args": ["--share=network"]
+			},
+			"sources": [
+				{
+					"type": "shell",
+					"commands": [ "Rscript -e \"install.packages(c('irlba', 'magrittr', 'pkgconfig', 'devtools'), repos='https://cloud.r-project.org',  Ncpus=4)\"" ]
+				},
+				{
+					"type":"git",
+					"url": "https://github.com/igraph/rigraph.git"
+				},
+				{
+					"type": "shell",
+					"commands": 
+						[ 
+							"echo \"all:\n\t\n\" > Makefile", 
+							"echo \"install:\n\tR CMD INSTALL ./ --configure-args=--disable-graphml\" >> Makefile" 
+
+						]
+				}
+			]
+		},
+		{
+			"name": "BDgraph",
+			"build-options": {
+				"build-args": ["--share=network"]
```

Comment:
network build isn't available within flathub - everything has to be included in the manifest and stable for reproducibility.